describe.js

```diff
@@ -87,6 +87,13 @@
 					e = new Error("Expected error '"+errorExpected+"' but got "+e);
 				}
 			}
+			if (testFailExpected) {
```

Comment:
The conditionals for `if (errorExpected)` and `if (testFailExpected)` seem really similar...  What's the difference?